src/Http/Session/DatabaseSession.php

```diff
@@ -164,8 +168,10 @@ public function write($id, $data): bool
      */
     public function destroy($id): bool
     {
+        /** @var string $pkField */
+        $pkField = $this->_table->getPrimaryKey();
```

Comment:
What harm would a string cast do here? Seems like we'd be able to skip the annotation and we'd get better runtime errors.

Reply:
I did try casting first but then psalm complained about potential invalid array to string cast :(